Include dot notation limitation (#5038)
segmentio/segment-docs: src/connections/destinations/actions.md

> info "Unsupported Special Characters"
> Mappings do not support the use of double quotes " or a tilde ~ in the trigger fields.

>info "Limitations"

> >Mapping fields do not support dot notation. i.e. properties.amount.cost or properties_amount.cost.

Reviewer: ```suggestion
> Mapping fields don't support dot notation. For example, properties.amount.cost or properties_amount.cost aren't supported.
```
